### ISSUE TITLE
Possibility to set ExecutionLevel for nsis portable 

### ISSUE THREAD

**Issue by marqu3z** [electron-userland/electron-builder#1440]
Hi, right now it is not possible to distribute portable app that needs admin privilege.
The only way is monkeypatching the portable template here: https://github.com/electron-userland/electron-builder/blob/master/packages/electron-builder/templates/nsis/portable.nsi#L5

As far as i see it is not even possible to use a custom script for portable app because if the target is `portable` the `script` option will not be considered.
https://github.com/electron-userland/electron-builder/blob/master/packages/electron-builder/src/targets/nsis.ts#L178

Is this something that could be fixed or is there another way i'm not aware of?

**Comment by develar** [comment]
Hmm... Interesting. Are you sure that your app needs admin privilege for all run time? I strongly recommend you to use UAC when admin rights is required. Please see https://github.com/automation-stack/electron-sudo

**Comment by marqu3z** [comment]
Hi @develar 
Yes we deeply investigated the alternatives.
In our case the app needs to change networks settings and restore them, and it can happen multiple times. Using electron-sudo every time the app needs to execute one of those network command the password prompt will show up.
This cause a very confusing and bad user experience, so we decided to drastically make the app run on each platform with admin privilege.
While for the other platform as a bit trickier because we had to patch the app after packing, for windows it seems just a matter of an option.
